dwst/scripts/test/particles.js

```diff
@@ -212,12 +212,139 @@ describe('particles module', () => {
         '${foo)}',
         '${foo()',
         '\\a',
+        'a\\',
       ];
       invalidParticlesExamples.forEach(invalidExample => {
         expect(() => {
           return parseParticles(invalidExample);
         }).to.throw(InvalidParticles);
       });
     });
+    it('should throw an InvalidParticles with correct errorparameters for only expression start', () => {
```

Comment:
You could probably remove *with correct errorparameters*. Even more so if we remove the combined test case as redundant.